lib/solarwinds_apm/opentelemetry/solarwinds_propagator.rb

```diff
@@ -80,14 +83,8 @@ def inject(carrier, context: ::OpenTelemetry::Context.current,
             end
             setter.set(carrier, TRACESTATE_HEADER_NAME, Utils.trace_state_header(trace_state))
           end
-        end
-
-        # Returns the predefined propagation fields. If your carrier is reused, you
-        # should delete the fields returned by this method before calling +inject+.
-        #
-        # @return [Array<String>] a list of fields that will be used by this propagator.
-        def fields
-          TRACESTATE_HEADER_NAME
```

Comment:
Do we still need `fields` for Ruby propagator definition?

Reply:
Yeah, after double checking with upstream, I think this fields is still needed.